New UniqueTokens card design and layout!

mirshko opened the pull request:
### PR Checklist (check all)

* [ ] Updated `CHANGELOG.md` to describe the included fixes and changes made
* [ ] Included issue number on the description of the PR
* [ ] Commented on the relevant issue thread about this PR

### Issue number

<!--- Which issue(s) is this PR targeting -->

### Changelog

<!--- Paste here the added change logs added to the CHANGELOG.md -->

mirshko committed:
:lipstick: reworking column layout of the unique tokens

mirshko committed:
:lipstick: update unique-token style

mirshko committed:
:speech_balloon: adding title attr to asset name for hover to see full name of asset

mirshko committed:
:lipstick: make sure border-radius clips the token card

mirshko commented:
<img width="1220" alt="Screenshot 2019-06-01 22 48 07" src="https://user-images.githubusercontent.com/8116716/58753538-c8dc9f80-84c0-11e9-8e83-1cff52e46887.png">

mirshko commented:
Its also responsive using css multicolumn!

ricburton commented:
🙌 Preview this pull request.

Built with commit f472dbf7804e9eb6b26b6d08f9040d19fd7818c6

https://deploy-preview-488--balance-manager.netlify.com

jackjdleslie reviewed (approved):
Looks good! No functional changes so will be fine to merge

mirshko committed:
:sparkles: move to css grid for tokens

will work on masonry or all same size cards soon

Commit:
Added support for empty NFTs

Uses OpenSea placeholder as NFT image, and uses contract name and NFT id for as NFT name
